avr/cores/megacommand/MCL/ExtSeqTrack.cpp

```diff
@@ -16,57 +16,239 @@ void ExtSeqTrack::set_speed(uint8_t _speed) {
 
 void ExtSeqTrack::set_length(uint8_t len) {
   length = len;
-  if (step_count >= length) {
+  while (step_count >= length && length > 0) {
```

Comment:
why would a negative value be passed in here?

Reply:
prevent lockup if length == 0.